lib/unison-util-file-embed/package.yaml

```diff
@@ -0,0 +1,34 @@
+name: unison-util-file-embed
+github: unisonweb/unison
+copyright: Copyright (C) 2013-2023 Unison Computing, PBC and contributors
+
+library:
+  source-dirs: src
+  when:
+    - condition: false
+      other-modules: Paths_unison_util_file_embed
+
+dependencies:
+  - base
+  - file-embed
+  - template-haskell
+
+ghc-options:
+  -Wall
+
+default-extensions:
+  - ApplicativeDo
+  - BlockArguments
+  - DeriveFunctor
+  - DerivingStrategies
+  - DoAndIfThenElse
+  - FlexibleContexts
+  - FlexibleInstances
+  - ImportQualifiedPost
+  - LambdaCase
+  - MultiParamTypeClasses
+  - NamedFieldPuns
+  - ScopedTypeVariables
+  - TupleSections
+  - TypeApplications
+  - ViewPatterns
```

Comment:
I just copied these from another package

Reply:
Absolutely we are (what would be the benefit of not? Lol)